### ISSUE TITLE
None of my stats are tracking

### ISSUE THREAD

**Issue by mrsilverhawk** [ShootMe/FallGuysStats#128]
I got it installed correctly but nothing is tracking. I have it set to my d drive where fall guys is located. Its acting like nothing is working on it except it counts correctly what round it is on. It will say finished 1st 0:00 and say the correct round but no times are saved.
![image](https://user-images.githubusercontent.com/54225128/176980207-9c35e0d6-042a-4524-977b-93facb677132.png)

**Comment by amagus** [comment]
Please share more details. 
Have you installed the game from the Epic Game Store or from Steam? 
What is your operating system version?
Have you messed with any settings? If yes, which ones have you changed.

**Comment by mrsilverhawk** [comment]
> 

The game is installed on steam on my D drive. Im using windows 10. I have not changed any settings. The rounds show itself correct but all timers dont work. They are all at 0 always.

**Comment by amagus** [comment]
What do you mean by correct? Where the game was installed from? Epic or Steam? The drive your game is installed make no difference as the logs are not stored there (they go to your application data directory, and the stats should work fine as long as those files can be accessed by the application). Can you please share screenshots of your settings panel? Is your windows user an administrator of the machine or/and is there any restriction on your user?

**Comment by mrsilverhawk** [comment]
I am the admin of my account, no restrictions of my user.
![image](https://user-images.githubusercontent.com/54225128/177076667-3195dc37-97e2-4eb9-bca9-c14015eb5c85.png)
![image](https://user-images.githubusercontent.com/54225128/177076683-a5b0e525-bef0-4df7-b3a2-d5ea05eb5c41.png)

**Comment by amagus** [comment]
If the overlay is displaying data correctly it means you haven’t installed the FallGuysStats correctly. (It’s unable to save the data on the builtin database). Download the ZIP file and extract ALL THE FILES it somewhere. DO NOT RUN IT DIRECTLY FROM THE ZIP FILE. Always extract to another folder snd try again.

**Comment by mrsilverhawk** [comment]
Ive done that already

**Comment by amagus** [comment]
the results will only post to the result screen if you see the rewards screen, can you confirm you played until you reached the rewards screen?

**Comment by mrsilverhawk** [comment]
yes I have played until the rewards screen, no timers work.

**Comment by amagus** [comment]
Can you share your Player.log so that I can see if I can reproduce the issue?

**Comment by mrsilverhawk** [comment]
where do I find player.log? I have these 
![image](https://user-images.githubusercontent.com/54225128/177247390-d6045e80-c15d-4111-9660-23d1b7754670.png)

**Comment by mrsilverhawk** [comment]
[Player.log](https://github.com/ShootMe/FallGuysStats/files/9043144/Player.log)

**Comment by abhie481** [comment]
Had the same issue. Overlay can detect the rounds currently in, ping, and position i finished. Other than that it shows 0 (timer also appear after finished, incorrectly). App settings is default. Tried run as admin, adding Log Path, launch the game via app, launch the app while game is running, none of them work. Here's my [Player.log](https://github.com/ShootMe/FallGuysStats/files/9043439/Player.log)

**Comment by amagus** [comment]
I managed to understand the issue now. Unsure why but your Fall Guys client is printing the timing in the logs using a different format, and it breaks the logger.
Example of log line we see on your Player.log files:

> 04.47.01.554: [GlobalGameStateClient].PreStart called at 2022-07-05T04:47:01  UTC

Usual format is this one:

> 12:31:56.407: [GlobalGameStateClient].PreStart called at 2022-07-04T12:31:56  UTC

I submitted a fix to my fork, allowing the application to accept both formats. 
I will do a pull request soon to this repo soon.
